tests/drivers/adc/adc_api/overlay-imx.conf

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) 2025, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
```

Comment:
need to update testcase.yaml to use this conf file or move it to boards subdirectory and rename it to be platform name .conf

Reply:
fixed